### ISSUE TITLE
Usage of PHP method from trait is incorrectly detected

### ISSUE THREAD

**Issue by mabar** [mesour-intellij-plugins/latte-pro-issues#6]
**Environment:**
 - PhpStorm version [e.g. 2022.2]
 - Plugin version [e.g. 1.0.4]

I am calling an method on enum case and can click through from Latte to PHP
![image](https://user-images.githubusercontent.com/20974277/183977639-9a9b18e4-4e8a-474a-84b4-20347351df28.png)

But when I click through from PHP to Latte, it finds different place of usage (DataGrid.latte instead of Alarm.latte from first screenshot)
![image](https://user-images.githubusercontent.com/20974277/183977786-4b88277a-eaf7-4058-8424-fdb0f0fe5293.png)

But the Latte file it found should not be found. Code it found is WIP and property on which is method called is undefined (now it looks like it found the method based on name, not on type)
![image](https://user-images.githubusercontent.com/20974277/183977989-40f72b78-8fca-46d0-8722-f3f7fe2f7f9b.png)

Also there are several other usages of this method in Latte files and I click from them to PHP, but from PHP view no Latte usages were found
![image](https://user-images.githubusercontent.com/20974277/183978239-d5c83bbc-af09-4254-9ab3-984d19fbc9b3.png)

**Comment by mesour** [comment]
@mabar this is working for me in `1.0.4`. I think, it sill also work in `1.0.3`. If it not working in `1.0.3`, wait and test it in `1.0.4`. Thanks for report 👍 After you test it, please leave comment here.

**Comment by mabar** [comment]
@mesour Got some time to test this properly and it is caused by trait usage.  Other references work fine. Here is minimal reproduction code:

```php
<?php

class TestClass
{
	use TestTrait;
}

trait TestTrait
{
	public function getLabel(): string
	{
		return 'foo';
	}
}
```

```latte
{varType \TestClass $shouldWork}
{$shouldWork->getLabel()}
```

You should be able to get from `TestTrait->getLabel()` to `$shouldWork` in Latte file.

**Comment by mesour** [comment]
@mabar it will be fixed in `1.0.5`
